README.md

```diff
@@ -64,6 +65,7 @@ Please see [Contributing to WebAssembly](https://github.com/WebAssembly/design/b
 [ecmascript_module_integration]: https://github.com/webassembly/esm-integration
 [exception_handling]: https://github.com/WebAssembly/exception-handling/blob/master/proposals/Exceptions.md
 [fixed-width_simd]: https://github.com/webassembly/simd/blob/master/proposals/simd/SIMD.md
+[function_references]: https://github.com/WebAssembly/function_references
```

Comment:
404 on https://github.com/WebAssembly/function_references
Might be https://github.com/WebAssembly/function-references

Reply:
Indeed, thanks!